scipy/stats/_binned_statistic.py

```diff
@@ -459,6 +459,43 @@ def binned_statistic_dd(sample, values, statistic='mean',
 
     .. versionadded:: 0.11.0
 
+    Examples
+    --------
+    >>> from scipy import stats
+    >>> import matplotlib.pyplot as plt
+    >>> from mpl_toolkits.mplot3d import Axes3D
+
+    Take 600 2-D arrays as an example. `binned_statistic_dd` can handle arrays
+    of higher dimension `D`. But a plot of dimension `D+1` is required.
+
+    >>> mu = np.array([0., 1.])
+    >>> Sigma = np.array([[1., -0.5],[-0.5, 1.5]])
```

Comment:
minor: could you use lowercase variable names? `N` is fine, but `sigma`, `x`, `y` would be nice. And `F` --> `multinormal` (or similar, a bit more descriptive).